doc/index.rst

```diff
@@ -1,224 +1,42 @@
 :orphan:
 
-.. title:: Matplotlib: Python plotting
+.. title:: Matplotlib Documentation
 
 .. module:: matplotlib
 
-Matplotlib: Visualization with Python
--------------------------------------
+Matplotlib Documentation 
+------------------------
 
-Matplotlib is a comprehensive library for creating static, animated,
-and interactive visualizations in Python.
-
-.. raw:: html
-
-   <div class="responsive_screenshots">
-      <a href="tutorials/introductory/sample_plots.html">
-         <div class="responsive_subfig">
-         <img align="middle" src="_images/sphx_glr_membrane_thumb.png"
-          border="0" alt="screenshots"/>
-         </div>
-         <div class="responsive_subfig">
-         <img align="middle" src="_images/sphx_glr_histogram_thumb.png"
-          border="0" alt="screenshots"/>
-         </div>
-         <div class="responsive_subfig">
-         <img align="middle" src="_images/sphx_glr_contour_thumb.png"
-          border="0" alt="screenshots"/>
-         </div>
-         <div class="responsive_subfig">
-         <img align="middle" src="_images/sphx_glr_3D_thumb.png"
-          border="0" alt="screenshots"/>
-         </div>
-      </a>
-   </div>
-   <span class="clear_screenshots"></span>
+Welcome to the documentation for Matplotlib |release|.
```

Comment:
I like the way pandas is doing this. More information and less distracting words:

![grafik](https://user-images.githubusercontent.com/2836374/126896739-a990604d-c21e-4a7d-ae5d-2d5288177b08.png)

I'd switch the order to version first, because I think it's the more important information.